core/src/test/java/org/infinispan/test/fwk/TestCacheManagerFactory.java

```diff
@@ -326,13 +328,9 @@ public static EmbeddedCacheManager createCacheManager(SerializationContextInitia
          globalBuilder.serialization().addContextInitializer(sci);
 
       ConfigurationBuilder builder = new ConfigurationBuilder();
-      builder
-            .clustering()
-            .cacheMode(mode)
-            .indexing()
-            .index(indexing ? Index.ALL : Index.NONE)
-            .addProperty("lucene_version", "LUCENE_CURRENT")
-      ;
+      builder.clustering().cacheMode(mode)
+            .indexing().index(indexing ? Index.ALL : Index.NONE)
```

Comment:
Nitpicking: Please enable "align when multiline" in IntelliJ for chained method calls, method call arguments, and method declaration parameters, I've gotten used to it and I like it a lot more than fixed indents nowadays.

Reply:
no way :)